src/stock_str.rs

```diff
@@ -424,6 +424,11 @@ pub enum StockMessage {
         fallback = "⚠️ Your email provider %1$s requires end-to-end encryption which is not setup yet."
     ))]
     InvalidUnencryptedMail = 174,
+
+    #[strum(props(
+        fallback = "Got outgoing message(s) encrypted for another setup. It seems you are using Delta Chat on multiple devices, but they cannot decrypt each other messages. You can use \"Add as Second Device\" feature to fix this."
```

Comment:
i tried to make it a bit shorter and also tried more on-point actions for fixing:

```suggestion
        fallback = "⚠️ It seems you are using Delta Chat on multiple devices, that cannot decrypt each others outgoing messages. To fix this, use "Settings / Add Second Device" on the older Device and follow the instructions."
```

the string may be overwritten in the UI anyways, however, it is good to have a nice fallback/pattern as well

Reply:
Applied your suggestion with minor edits. I think "older device" is better than "first device", particularly if the user already has two or more devices